index.js

```diff
@@ -54,6 +57,11 @@ var hasOwnProperty = Object.prototype.hasOwnProperty || function (obj, key) {
 	return key in obj;
 };
 
+function isWritable(object, key) {
+	var descriptor = Object.getOwnPropertyDescriptor(object, key);
```

Comment:
this will fail on ES3 engines; we'll need to use `gopd` here, and check it conditionally.